test/e2e/examples.go

```diff
@@ -197,31 +197,36 @@ var _ = Describe("[Feature:Example]", func() {
 			controllerYaml := mkpath("cassandra-controller.yaml")
 			nsFlag := fmt.Sprintf("--namespace=%v", ns)
 
-			By("starting service and pod")
+			By("Starting the cassandra service and pod")
 			runKubectlOrDie("create", "-f", serviceYaml, nsFlag)
 			runKubectlOrDie("create", "-f", podYaml, nsFlag)
+
+			Logf("waiting for first cassandra pod")
 			err := waitForPodRunningInNamespace(c, "cassandra", ns)
 			Expect(err).NotTo(HaveOccurred())
 
+			Logf("waiting for thrift listener online")
 			_, err = lookForStringInLog(ns, "cassandra", "cassandra", "Listening for thrift clients", serverStartTimeout)
 			Expect(err).NotTo(HaveOccurred())
 
+			Logf("wait for service")
 			err = waitForEndpoint(c, ns, "cassandra")
 			Expect(err).NotTo(HaveOccurred())
 
-			By("create and scale rc")
+			// Create an RC with n nodes in it.  Each node will then be verified.
```

Comment:
it is only meaningful in context of the diff, because we don't (create, scale to n, verify), but rather (create n, verify).  agree its not super useful, it's at best redundant since this can be seen from the code .